src/component/toolbox/feature/DataView.ts

```diff
@@ -318,32 +318,40 @@ class DataView extends ToolboxFeature<ToolboxDataViewFeatureOption> {
     private _dom: HTMLDivElement;
 
     onclick(ecModel: GlobalModel, api: ExtensionAPI) {
+        // FIXME: better way?
+        setTimeout(() => {
```

Comment:
Is it necessary to use `setTimeout`? The `tooltip` should have been displayed before `onclick` event happens. I'm not sure if there is edge cases or something else I didn't thought of.

Reply:
The tooltip won't disappear if no delay. I don't know why yet.